portfolio/src/main/webapp/comment.js

```diff
@@ -12,27 +12,35 @@ function getLoginStatus() {
         let htmlText = json.htmlText;
 
         if (!isLoggedIn) {
-            const loginDiv = document.createElement("div");
-            let id = document.createAttribute("id");
-            id.value = "login";
-            loginDiv.setAttributeNode(id);
-            loginDiv.innerHTML = htmlText;
+            let loginDiv = createLoggingSection("login", htmlText);
 
             let body = document.getElementById("body");
             body.appendChild(loginDiv);
         } else {
-            const logoutDiv = document.createElement("div");
-            let id = document.createAttribute("id");
-            id.value = "logout";
-            logoutDiv.setAttributeNode(id);
-            logoutDiv.innerHTML = htmlText;
+            let logoutDiv = createLoggingSection("logout", htmlText);
```

Comment:
Isn't this confusing? create login section named logout?

Reply:
I will rename to `createLoggingRelatedSection`, logging not log in:)